src/controllers/itemsbysupermarket.js

```diff
@@ -1,3 +1,13 @@
+const query = require('../model/queries/breakdownQuery').breakdownQuery
 exports.get = (req, res) => {
-    res.render('itemsbysupermarket');
+    let productIds = req.query.product;
+    productIds = typeof productIds === 'string' ? productIds = [productIds] : productIds;
```

Comment:
Thought this line may be clearer

`productIds = typeof productIds === 'string' ? [productIds] : productIds;`